src/audio/lib/load.c

```diff
@@ -1133,9 +1133,19 @@ void AudioLoad_Init(void* heap, u32 heapSize) {
         s32 i;
         u8* audioContextPtr = (u8*)&gAudioCtx;
 
+#ifndef AVOID_UB
+        //! @bug This clearing loop sets one extra byte to 0 following gAudioCtx.
+        //! In practice this is harmless as it would set the most significant byte in gAudioCustomUpdateFunction to 0,
+        //! which was just reset to NULL above.
         for (i = sizeof(gAudioCtx); i >= 0; i--) {
             *audioContextPtr++ = 0;
         }
+#else
+        // Avoid out-of-bounds variable access
+        for (i = sizeof(gAudioCtx); i > 0; i--) {
+            *audioContextPtr++ = 0;
+        }
```

Comment:
Why not `bzero` instead?
```suggestion
        bzero(&gAudioCtx, sizeof(gAudioCtx));
```

Reply:
I wanted to keep the behavioral differences minimal between the two. If more people are fine with larger departures from the original in AVOID_UB blocks I can change it.

Reply:
I also prefer minimal difference. calling a function or fundamentally different code wouldn't be in the spirit of avoid_ub "replicate the undefined behavior as it behaves in vanilla but without undefined behavior"